tests/integration-tests.bats

```diff
@@ -33,5 +33,11 @@ load ../script/libtest
 @test "Test function: post-nodejs" {
   test_kubeless_function post-nodejs
 }
-
+@test "Test function: pubsub" {
+  skip "XXX(jjo): flaky kafka->zookeeper under minikube, likely from its kube-dns"
```

Comment:
In fact it had happened before we landed the kafka/zk statefulset setup (./manifests/kafka & ./manifests/zookeeper), and now it's running pretty well on minikube. I think we should hold this PR a bit to figure out what causes the DNS problem. It's better to have the pubsub test executed